client/service/Cargo.toml

```diff
@@ -81,6 +82,9 @@ async-trait = "0.1.50"
 tokio = { version = "1.15", features = ["time", "rt-multi-thread", "parking_lot"] }
 tempfile = "3.1.0"
 directories = "4.0.1"
+regex = "1"
+libc = "0.2"
+nix = "0.23"
```

Comment:
Looks like `nix` is not used (yet?)

Reply:
Ah, good catch! I originally used `nix` in an earlier version of this code and forgot to remove it from the `Cargo.toml` after it wasn't needed anymore.